### PR TITLE
upgrade config version to v2beta10

### PR DESCRIPTION
**Pull request by anshlykov**
Prepping for #5009

### CODE REVIEW

File: pkg/skaffold/schema/v2beta9/upgrade_test.go [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+	"testing"
+
+	yaml "gopkg.in/yaml.v2"
+
+	next "github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/latest"
+	"github.com/GoogleContainerTools/skaffold/testutil"
+)
+
+func TestUpgrade(t *testing.T) {
+	yaml := `apiVersion: skaffold/v2beta9
```
Comment (tejal29) [inline pkg/skaffold/schema/v2beta9/upgrade_test.go:29]: Please add a `port-forward` config with int port so in next PR, you can verify it gets updated correctly.

File: pkg/skaffold/schema/v2beta9/upgrade.go [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/util"
+	pkgutil "github.com/GoogleContainerTools/skaffold/pkg/skaffold/util"
+)
+
+// Upgrade upgrades a configuration to the next version.
+// Config changes from v2beta9 to v2beta10
+// 1. No additions:
+// 2. No removals
+// 3. Updates:
+//    - sync.auto becomes boolean
```
Comment (tejal29) [inline pkg/skaffold/schema/v2beta9/upgrade.go:30]: Remove updates and update in #5009 
```suggestion
```

**Review by tejal29 — APPROVED** [review]
Please address nits.

File: pkg/skaffold/schema/v2beta9/upgrade_test.go (hunk 1 above)
Comment (anshlykov) [reply pkg/skaffold/schema/v2beta9/upgrade_test.go:29]: I will make this change in #5009

File: pkg/skaffold/schema/v2beta9/upgrade.go (hunk 2 above)
Comment (anshlykov) [reply pkg/skaffold/schema/v2beta9/upgrade.go]: thanks, fixed

**Review by briandealwis — APPROVED** [review]
Thanks!

**Pull request merged**